src/mawpy/workflow1.py

```diff
@@ -19,15 +23,16 @@ def workflow1(
     duration_constraint1,
     duration_constraint2,
 ):
-    IC(input_file, output_file, spatial_constraint, duration_constraint1)
+    incremental_clustering(input_file, output_file, spatial_constraint, duration_constraint1)

```

Comment:
I understand that the new module cannot be fully tested yet. But in that case I would not change this line to use the new module since it has not been verified. In other words, it's ok to add the new `incremental_clustering` module, but let's defer making the invocation.

Reply:
I have reverted the changes! Going forward, I plan to keep the updated workflows in a folder called 'workflows'